keel-docker/src/test/kotlin/com/netflix/spinnaker/keel/docker/SampleDockerImageResolverTests.kt

```diff
@@ -58,13 +58,20 @@ class SampleDockerImageResolverTests : JUnit5Minutests {
     ),
     account = "test"
   )
+
+  private val nullable = SampleSpecWithContainer(
```

Comment:
Suggestion for clarity:
```suggestion
  private val nullContainerSpec = SampleSpecWithContainer(
```